proto/buf.yaml

```diff
@@ -34,6 +34,7 @@ lint:
     # https://github.com/twitchtv/twirp/issues/70#issuecomment-470367807
     UNARY_RPC:
       - teleport/lib/teleterm/v1/service.proto
+      - accessgraph/v1alpha/query.proto
```

Comment:
Do consider the warning above. Streams come with various difficulties and limitations, if the unary would do then it's likely the best long-term choice.

Reply:
Yes, we had very long discussion about it. For now, we want to stick with the stream. If it's problematic then we can think about a replacement.